apps/chat/src/components/Chatbar/Conversation.tsx

```diff
@@ -775,7 +792,7 @@ export const ConversationComponent = ({
           ref={refs.setFloating}
           {...getFloatingProps()}
           className={classNames(
-            'absolute right-3 z-50 flex cursor-pointer justify-end group-hover/conversation-item:visible',
+            'absolute right-3 z-50 flex cursor-pointer justify-end group-hover:visible',
```

Comment:
Are you sure that hover logic isn't broken?

Reply:
I checked and I don't think it's breaking anything.
BG of `<ShareIcon />` was broken because of the different groups.